source/real_locations.rst

```diff
@@ -0,0 +1,116 @@
+Checking the people in a location (SIR)
+=======================================
+
+What we were doing
+------------------
+
+While calculating the number of people who could potentially infect a person, we did the following:
+
+.. code-block:: scala
+
+    def computeInfectedFraction(node: Node, placeType: String, context: Context): Double = {
+    val totalNeighbourCount = node.getConnectionCount(node.getRelation[Person]().get)
+    if (totalNeighbourCount == 0)
+      return 0d
+    val infectedNeighbourCount = node.getConnectionCount(node.getRelation[Person]().get,
+      "infectionState" equ Infected).toDouble
+    infectedNeighbourCount / totalNeighbourCount.toDouble
+  }
+
+Let's take a closer look at the first line, how we calculate ``totalNeighbourCount``.
+
+.. code-block:: scala
+
+  val totalNeighbourCount = node.getConnectionCount(node.getRelation[Person]().get)
+
+Assume that the node was an ``Office``. In that case,
+
+.. code-block:: scala
+
+  node.getRelation[Person]().get
+
+returns the ``EMPLOYER_OF`` string. Now, ``totalNeighbourCount`` is counting the total number of nodes which the workplace node is the ``EMPLOYER_OF``.
+
+The problem arises when we introduce quarantining to our model. Someone who's infected may follow a different schedule, and stay at home. However, ``totalNeighbourCount`` *doesn't care* about the location a person has at a particular tick: all it does it count the number of people with the appropriate relations.
+
+As a result of this, quarantining individuals is far less effective than it should be. It still could have a minor effect (depending on your model), as someone who's infected does not go to their workplace. Regardless, that person still infects others currently at the workplace.
```

Comment:
No point going into efficacy here. It's sufficient to just say that the results would not be what the modeller intended, which is far more serious!